src/compas_viewer/scene/brepobject.py

```diff
@@ -5,14 +5,14 @@
 from .sceneobject import DataType
 
 try:
```

Comment:
Is it necessary to put the whole class definition into the try block here? Since it is alreadt inside try in the `__init__.py`

Reply:
deleted